public/index.css

```diff
@@ -0,0 +1,20 @@
+:root {
```

Comment:
I think this style should be applied to the `body` element. `:root` pseudo-class is a bit weird If you're absolutely sure that the `body` is not the right place for this style the `html` element as it's more readable than `:root` and is generally the same.

Reply:
its not much of a difference, but changing ...